doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ArchetypeManagement/proc-business-central-creating-archetype-project.adoc

```diff
@@ -16,7 +16,9 @@ You can use archetypes to create a project in {CENTRAL}. When you create a proje
 . Type the project name and description in the *Name* and *Description* fields respectively.
 . Click *Configure Advanced Options*.
 . Select the *Based on template* check box.
-. Select the archetype from drop-down options if required. The default archetype is selected that is already set in the space.
+. Select the archetype from drop-down options if required.
++
+The default archetype is selected that is already set in the space.
```

Comment:
```suggestion
The default archetype is already set in the space.
```
I assume that this means that the default archetype already set in the space is selected in the drop-down when the user clicks on the drop-down options. Am I correct? A little unclear. I have suggested a change but you could also remove the line altogether.

Reply:
Yes, that is correct!